test/TEST-20-MAINPIDGAMES/testsuite.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -ex
+set -o pipefail
+
+systemd-analyze set-log-level debug
+systemd-analyze set-log-target console
```

Comment:
Hopefully last nitpick - this will fail on RHEL7, as the `set-log-target` subcommand was introduced in v227 (https://github.com/systemd/systemd/commit/2ca2a91cf1deba83825692f1ce06116d2aed2379)

Reply:
Removed.